admin-site/src/App.svelte

```diff
@@ -52,23 +52,27 @@
     isOccasionOpen = value.detail.isOccasionOpen;
 
   }
+
+  function messageCloseEventForm(value){
+    openEventCreation = value.detail.openEventCreation;
+  }
+  function messageOpenEventForm(value){
+    openEventCreation = value.detail.openEventCreation;
+  }
+  function messageCloseOccasionForm(value){
+    openOccasionCreation = value.detail.openOccasionCreation;
+  }
+  function messageOpenOccasionForm(value){
+    openOccasionCreation = value.detail.openOccasionCreation;
+    
+  }
   
   function broadcastStatusFromBackButton(value){
```

Comment:
The purpose of this function is a bit opaque, it might benefit from a comment or a clearer name

Reply:
Ok, changed the name and added some comments. In general I'll comment more.